### ISSUE TITLE
Using tabs raises error TF20507

### ISSUE THREAD

**Issue by itsho** [XpiritBV/Vsts-Build-Task-InlinePowershell#19]
Hi.

thank you for your Task. it is very helpful.

This issue is slightly related to #13: 

When using tabs within the editor and trying to save - TFS throws an error 
> TF20507: The string argument contains a character that is not valid:'u0009'. Correct the argument, and then try the operation again.
> Parameter name: definition.Steps.Script

To reproduce it, copy the function containing the following line, and try to save
https://github.com/fakhrulhilal/TFSGitDownloader/blob/2c8b2668de78966871140b8bd4d0b2568338eaf9/GitDownloader/GitDownloader.ps1#L132

Thank you.

**Comment by pgroene** [comment]
Sorry for the late response, u0009 is a tab, but I do not have a clue why that would fail. Can you replace the tab with spaces?

**Comment by itsho** [comment]
Yeah, It works when using spaces instead of tabs. That's what I did eventually
